crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -125,6 +127,7 @@ impl Extension for LoggerExtension {
         let resp = next.run(ctx, operation_name).await;
         if resp.is_err() {
             for err in &resp.errors {
+                let error_code = &err.extensions.as_ref().and_then(|x| x.get("code"));
```

Comment:
Consider adding an UNKNOWN enum value and using that as a default.  Then you can use a match block much easer below instead of the "is it a code::INTERNALSERVER_ERROR else some other thing" logic.

https://github.com/MystenLabs/sui/blob/7bc4de5c00079aab15dc6b8aefaf1a6c78547156/crates/sui-graphql-rpc/src/error.rs#L14-L19

Reply:
Good idea, added!